Sources/SpeziOnboarding/OnboardingView.swift

```diff
@@ -208,6 +215,23 @@ public struct OnboardingView<TitleView: View, ContentView: View, ActionView: Vie
 }
 
 
+extension EnvironmentValues {
+    @Entry fileprivate var onboardingViewEdgesWithPaddingDisabled: Edge.Set = []
+}
+
+
+extension OnboardingView {
+    /// Disables the ``OnboardingView``'s implicit padding for the specified edges.
+    ///
+    /// If this modifier is applied multiple times, the outermost call will take precedence.
+    ///
+    /// - Note: If the ``OnboardingView`` is contained in an ``OnboardingStack``, its top edge will already be disabled implicitly.
+    public func disablePadding(_ edges: Edge.Set) -> some View {
+        self.environment(\.onboardingViewEdgesWithPaddingDisabled, edges)
+    }
+}
```

Comment:
I'm not quite sure about this entire top padding logic that we now have in the `OnboardingStack`. We didn't need that before, right? Why do we need it now?